### PR TITLE
[18.0][MIG] account_invoice_refund_link: Migration to 18.0 

### PR DESCRIPTION
**Pull request by peluko00**
Module migrated to version 18.0

cc https://github.com/APSL 163709
@miquelalzanillas @javierobcn @mpascuall @BernatObrador @ppyczko please review

- [Issue] https://github.com/OCA/account-invoicing/issues/1810

### CODE REVIEW

**Review by mpascuall — APPROVED** [review]
LGTM, tested in runboat

**Review by ppyczko — APPROVED** [review]
LGTM!

**Review by BernatObrador — APPROVED** [review]
LGTM

File: account_invoice_refund_link/README.rst [hunk 1]
```diff
@@ -90,6 +90,10 @@ Contributors
 
       -  Luis J. Salvatierra <luis.salvatierra@factorlibre.com>
 
+-  `Factor Libre <https://factorlibre.com>`__:
+
+      -  Luis J. Salvatierra <luis.salvatierra@factorlibre.com>
```
Comment (pedrobaeza) [inline account_invoice_refund_link/README.rst]: This contributor is being added at 18.0 migration commit, which doesn't sound correct.

Comment (peluko00) [reply account_invoice_refund_link/README.rst]: Fixed!

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**